apps/gallery/js/ImageEditor.js

```diff
@@ -1195,42 +1187,42 @@ ImageEditor.prototype.drawCropControls = function(handle) {
   context.beginPath();
```

Comment:
Answering my own question, I think that the canvas is using CSS pixels, so these lines widths are already implicitly multiplied by devicePixelRatio